index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
+<!-- saved from url=(0039)https://skycocker.github.io/chromebrew/ -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link rel="shortcut icon" href="brew.png"" type="image/png">
```

Comment:
Can these images be put in an `images` folder and referenced via `href="/images/brew.png"`?  You have an extra double quote in there also.

Reply:
fine

Reply:
whoops